bundles/net.tourbook/src/net/tourbook/ui/messages_fr.properties

```diff
@@ -552,14 +552,20 @@ Image_ToolTip_TourInfo_Hovered = tour-info-hovered.png
 Segmenter_Tooltip_Label_Altimeter           = Altim\u00E8tre
 Segmenter_Tooltip_Label_AltitudeDifference  = D\u00E9nivel\u00E9
 Segmenter_Tooltip_Label_ComputedTime_Break  = Temps \u00E0 l'arr\u00EAt
+Segmenter_Tooltip_Label_ComputedTime_Moving = Temps de d\u00E9placement
 Segmenter_Tooltip_Label_DeviceTime_Elapsed  = Temps \u00E9coul\u00E9
 Segmenter_Tooltip_Label_DeviceTime_Paused   = Temps de pause
 Segmenter_Tooltip_Label_DeviceTime_Recorded = Temps d'enregistrement
 Segmenter_Tooltip_Label_Distance            = Distance
 Segmenter_Tooltip_Label_Gradient            = Pente
-Segmenter_Tooltip_Label_ComputedTime_Moving = Temps de d\u00E9placement
 Segmenter_Tooltip_Label_Title               = Segment de parcours
 
+Statistic_Tooltip_Label_ColumnHeader_Month              = Total mensuel
+Statistic_Tooltip_Label_ColumnHeader_Percentage         = %
+Statistic_Tooltip_Label_ColumnHeader_Percentage_Tooltip = Pourcentage de la valeur du b\u00E2ton par rapport \u00E0 la valeur totale
+Statistic_Tooltip_Label_ColumnHeader_Year               = Total annuel
+Statistic_Tooltip_Label_NumberOfTours                   = Nombre de parcours
```

Comment:
@FJBDev 

You should wait with the translation of "my" new texts to avoid waisting of your time.

The development is a dynamic process and I already discarded 2 of these new texts. When a new version is done with the development then I do the german translation and during this phase it is also possible that I change the english text, after that, the texts are final.

You can find removed text keys with this tool (it takes a longer computing time) which I also use to cleanup unused text keys

![mt-unused-nls-keys](https://user-images.githubusercontent.com/1283445/95746830-bcad4880-0c97-11eb-8216-9d174af9482f.png)

![mt-unused-nls-keys-2](https://user-images.githubusercontent.com/1283445/95746833-bd45df00-0c97-11eb-848e-768de7d3af29.png)

Reply:
Good point, I will wait until it is ready to be translated.